server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/SectionNode.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree;
+
+import org.eclipse.lsp4j.Location;
+
+/**
+ * The class represents section context in COBOL.
+ */
+public class SectionNode extends Node {
```

Comment:
What if parent for SECTION,  Program?  Would not be better to add DIVISION (general one at least)?

Reply:
Now it is a program, you are right. But it's just because this kind is enough for the current task. Sure we must add DIVISION, but we can do it in other changes when we will use it. It can be added for Outline tree for example.